.storybook/helpers/elements.data.ts

```diff
@@ -5,130 +5,135 @@
-  { name: 'Rhenium', symbol: 'Re', number: 75, electronegativity: 1.9 },
-  { name: 'Rhodium', symbol: 'Rh', number: 45, electronegativity: 2.28 },
-  { name: 'Roentgenium', symbol: 'Rg', number: 111, electronegativity: 0 },
-  { name: 'Rubidium', symbol: 'Rb', number: 37, electronegativity: 0.82 },
-  { name: 'Ruthenium', symbol: 'Ru', number: 44, electronegativity: 2.2 },
-  { name: 'Rutherfordium', symbol: 'Rf', number: 104, electronegativity: 0 },
-  { name: 'Samarium', symbol: 'Sm', number: 62, electronegativity: 1.17 },
-  { name: 'Scandium', symbol: 'Sc', number: 21, electronegativity: 1.36 },
-  { name: 'Seaborgium', symbol: 'Sg', number: 106, electronegativity: 0 },
-  { name: 'Selenium', symbol: 'Se', number: 34, electronegativity: 2.55 },
-  { name: 'Silicon', symbol: 'Si', number: 14, electronegativity: 1.9 },
-  { name: 'Silver', symbol: 'Ag', number: 47, electronegativity: 1.93 },
-  { name: 'Sodium', symbol: 'Na', number: 11, electronegativity: 0.93 },
-  { name: 'Strontium', symbol: 'Sr', number: 38, electronegativity: 0.95 },
-  { name: 'Sulfur', symbol: 'S', number: 16, electronegativity: 2.58 },
-  { name: 'Tantalum', symbol: 'Ta', number: 73, electronegativity: 1.5 },
-  { name: 'Technetium', symbol: 'Tc', number: 43, electronegativity: 1.9 },
-  { name: 'Tellurium', symbol: 'Te', number: 52, electronegativity: 2.1 },
-  { name: 'Terbium', symbol: 'Tb', number: 65, electronegativity: 0 },
-  { name: 'Thallium', symbol: 'Tl', number: 81, electronegativity: 1.62 },
-  { name: 'Thorium', symbol: 'Th', number: 90, electronegativity: 1.3 },
-  { name: 'Thulium', symbol: 'Tm', number: 69, electronegativity: 1.25 },
-  { name: 'Tin', symbol: 'Sn', number: 50, electronegativity: 1.96 },
-  { name: 'Titanium', symbol: 'Ti', number: 22, electronegativity: 1.54 },
-  { name: 'Tungsten', symbol: 'W', number: 74, electronegativity: 2.36 },
-  { name: 'Ununbium', symbol: 'Uub', number: 112, electronegativity: 0 },
-  { name: 'Ununhexium', symbol: 'Uuh', number: 116, electronegativity: 0 },
-  { name: 'Ununpentium', symbol: 'Uup', number: 115, electronegativity: 0 },
-  { name: 'Ununquadium', symbol: 'Uuq', number: 114, electronegativity: 0 },
-  { name: 'Ununseptium', symbol: 'Uus', number: 117, electronegativity: 0 },
-  { name: 'Ununtrium', symbol: 'Uut', number: 113, electronegativity: 0 },
-  { name: 'Uranium', symbol: 'U', number: 92, electronegativity: 1.38 },
-  { name: 'Vanadium', symbol: 'V', number: 23, electronegativity: 1.63 },
-  { name: 'Xenon', symbol: 'Xe', number: 54, electronegativity: 2.6 },
-  { name: 'Ytterbium', symbol: 'Yb', number: 70, electronegativity: 0 },
-  { name: 'Yttrium', symbol: 'Y', number: 39, electronegativity: 1.22 },
-  { name: 'Zinc', symbol: 'Zn', number: 30, electronegativity: 1.65 },
-  { name: 'Zirconium', symbol: 'Zr', number: 40, electronegativity: 1.33 },
+  { expanded: false, name: 'Actinium', symbol: 'Ac', number: 89, electronegativity: 1.1 },
```

Comment:
As discussed this new optional field could omitted here, by just using "!!element?.expanded" when the new field is used. But it's just personal opinion - this means we are setting the default value to false (which maybe does not correspond to our need? - I am not sure about that)

Reply:
Corrected.